flask_security/views.py

```diff
@@ -45,6 +45,8 @@ def _render_json(form, include_user=True, include_auth_token=False):
         response = dict()
         if include_user:
             response['user'] = dict(id=str(form.user.id))
```

Comment:
Yes, you are right. I think is a better solution because we don't have the necessity to define another setting in the config. If you want a different payload you only need to rewrite the method get_security_payload() in the User Model. 

So, in the view.py file we get the payload:

```python
response = dict()
if include_user:
    response['user'] = form.user.get_security_payload()
```

```python
class UserMixin(BaseUserMixin):
    # ...
    def get_security_payload(self):
        """Serialize user object as response payload."""
        return {'id': str(self.id)}
```

If you need a different Payload redifine the method: 

```python
UserModel(Base):
    id = db.Column(db.Integer, primary_key=True)
    email = db.Column(db.String(120), unique=True, nullable=False)
    password = db.Column(db.String(255))
    name = db.Column(db.String(80))
    
    def get_security_payload(self):
        return {
            id: self.id,
            name: self.name,
            email: self.email  
        }
```